src/layoutBuilder.js

```diff
@@ -823,20 +1062,41 @@ LayoutBuilder.prototype.buildNextLine = function (textNode) {
 		isForceContinue = inline.noNewLine && !isHardWrap;
 	}
 
+	if (containsInlineRtl) {
+		// TODO Pretty sure that both of these RTL functions can be combined into one
+		// but in the interests of making changes non-breaking I'm not going to
+		// attempt it during this work
+		const styleStack = new StyleContextStack(
+			this.styleDictionary,
+			this.defaultStyle
+		);
+		styleStack.push(textNode);
+		styleStack.push({ font: 'NotoSansArabic' });
```

Comment:
It's a bit of a shame that this is hardcoded. Would it contain hebrew as well?

Reply:
I thought exactly the same - it seems odd to me that it's hardcoded here.

My plan is to build a combined font file of english, Hebrew and Arabic (NotoSansRTL or something) which I'll replace this with, but I have a slight concern that this line of the code will prohibit doing stuff like rendering other scripts (CJK etc) inline with RTL. This concern may prove to be baseless.

I haven't completely got my head around the `StyleContextStack` class here and what's going on with the `styleDictionary`.

Reply:
Fixed in the more recent work